addons/web_editor/models/ir_qweb.py

```diff
@@ -47,10 +47,25 @@ class QWeb(models.AbstractModel):
 
     # compile directives
 
+    def _compile_node(self, el, options):
+        if options.get('is-snippet') == options['template']:
```

Comment:
Not sure why `is-snippet` is not a boolean? Why do you need to compare it to `options['template']`?

Reply:
The `t-call-options` are passed to the t-called templates so all t`-call` inside of a snippet will trigger this if i didn't check.

Reply:
Rename is-snippet by snippet-key, it is hard for me to read "if is snippet equals template"